### PR TITLE
Updated enchant to pull movement out

### PR DESCRIPTION
**Pull request by dr-leustyin**

### CODE REVIEW

File: craft.lic [hunk 1]
```diff
@@ -144,6 +144,7 @@ class Craft
       order_item(crafting_data['artificing'][@hometown]['stock-room'], 2)
       order_item(crafting_data['artificing'][@hometown]['stock-room'], 6)
       stow_hands
+      walk_to(@training_room)
```
Comment (Sarvatt) [inline craft.lic]: Why remove this from craft.lic? You only want to move the movement logic into workorders right? Craft does movement  on its own for every other craft

File: enchant.lic [hunk 2]
```diff
@@ -59,10 +59,6 @@ class Enchant
     @equipment_manager = EquipmentManager.new
     @equipment_manager.empty_hands
 
-    if @use_own_brazier
```
Comment (rpherbig) [inline enchant.lic]: 💡 Can we remove the `DRCT` references from this script now?

File: craft.lic (hunk 1 above)
Comment (rpherbig) [inline craft.lic]: 💡 It really feels like we ought to have a helper method for this

**Comment by rpherbig** [comment]
@dr-leustyin my only concern is that someone might be using `enchant` in their YAML when now they should be using `craft`. Are you cool with monitoring LNet for a bit after this is merged?

File: enchant.lic (hunk 2 above)
Comment (dr-leustyin) [reply enchant.lic]: Done!

File: craft.lic (hunk 1 above)
Comment (dr-leustyin) [reply craft.lic]: Any suggestions?  walk_to in itself is a helper method

**Pull request merged**